server/graphql/v2/mutation/TierMutations.js

```diff
@@ -0,0 +1,112 @@
+import { GraphQLNonNull } from 'graphql';
+
+import models from '../../../models';
+import { checkRemoteUserCanUseAccount } from '../../common/scope-check';
+import { NotFound, Unauthorized } from '../../errors';
+import { getIntervalFromTierFrequency } from '../enum/TierFrequency';
+import { idDecode } from '../identifiers';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { getValueInCentsFromAmountInput } from '../input/AmountInput';
+import { TierCreateInput } from '../input/TierCreateInput';
+import { fetchTierWithReference, TierReferenceInput } from '../input/TierReferenceInput';
+import { TierUpdateInput } from '../input/TierUpdateInput';
+import { Tier } from '../object/Tier';
+
+const tierMutations = {
+  editTier: {
+    type: new GraphQLNonNull(Tier),
+    description: 'Edit a tier.',
+    args: {
+      tier: {
+        type: new GraphQLNonNull(TierUpdateInput),
+      },
+    },
+    async resolve(_, args, req) {
+      checkRemoteUserCanUseAccount(req);
+
+      const tierId = idDecode(args.tier.id);
```

Comment:
It currently makes no difference, but ideally, all IDs should be typed to make their order non-predictable 

```suggestion
      const tierId = idDecode(args.tier.id, 'tier');
```

The value could also be added to `IDENTIFIER_TYPES`.